CONTRIBUTING.md

```diff
@@ -237,13 +237,10 @@ pip install -U pip
 pip install -r requirements.txt
 ```
 
-The first command ensures you are using an up to date version of `pip`,
+The first command ensures you are using an up-to-date version of `pip`,
 and the second command installs all dependencies.
-The third command ensures to install the BIDS schema code as an "editable" install,
-so that if you make changes to the schema files,
-these are automatically reflected in the sourcecode.
```

Comment:
indeed, it seems that the next paragraph points to that "editable" install. and there is no 3rd command there, which seems was removed in #1266 but this text was not adjusted.